### PR TITLE
Add user guide placeholder page and "learn" menu dropdown

### LINKED ISSUE #681: Ensure nav links display active styles on subpaths
As described by @smwingo in slack:
> noticing that if i click on "explore" and it defaults to the "campaign" part of explore page, the "explore" word in the top menu stays bold/underlined....but if i click for any of the other "explore" pages (platforms, instruments, data prods) that top "explore" word is in same text as rest of top menu

**Campaigns**
image

**Data Products**
image

### Solution:
The logic for determining "Explore" active styles should include subpaths.

### PR DESCRIPTION
**Pull request by LanesGood**
Adds user guide placeholder page and "Learn" menu dropdown

Screenshot 2024-08-28 at 4 49 18 PM

contributes to #674. Closes #681

### CODE REVIEW

**Comment by LanesGood** [comment]
@willemarcel as this contains fixes for some outstanding mobile nav issues, I decided to hide the user guide page and merge in your branch #683. Can you review and we'll add the user guide data later on when ready?

File: src/pages/user-guide.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Layout, { PageBody, SectionContent } from "../components/layout"
+import SEO from "../components/seo"
+
+export default function UserGuide() {
+  return (
+    <Layout>
```
Comment (willemarcel) [inline src/pages/user-guide.js]: This `<Layout>` component is duplicating the header

**Review by willemarcel — APPROVED** [review]
I've left a comment in the user guide page, but the other parts are working well

File: src/pages/user-guide.js (hunk 1 above)
Comment (LanesGood) [reply src/pages/user-guide.js]: Thanks for the catch Wille! Removed

**Pull request merged**